### ISSUE TITLE
How to draw a loss function curve

### ISSUE THREAD

**Issue by zhangth971** [advimman/lama#313]
I want to see the loss function curve of the training process, is there any way to do it
